report/3.Theory/LanguageSpecification.tex

```diff
@@ -104,12 +132,17 @@ \subsection{Functions}
 \textbf{Grammar for function calls}\\
 Function calls are done with infix notation, meaning that if a given function is defined with parameters, the first argument has to appear before the function identifier, while all remaining arguments must appear after, in sequence.
 
-The following examples showcase this:
-\begin{itemize}
-	\item \texttt{printHelloWorld}
-	\item \texttt{"Hello World !!" print}
-	\item \texttt{"Hello" printWords "World" "!!"}
-\end{itemize}
+The following listing showcase this.
+\begin{lstlisting}[language=HCL,caption={Function call notation in HCL.
+printHelloWolrd, print and printWords are functions and have been declared before hand.
```

Comment:
that text is a bit weird. maybe something like "Function call notation in HCL.
printHelloWolrd, print and printWords are example functions calls". det er implicit er det der declared beforehand og ikke afterwards. fordi ellers var det en undeclared fejl :P

Reply:
Fixed